src/couch_mrview/include/couch_mrview.hrl

```diff
@@ -70,7 +70,9 @@
 
     direction = fwd,
     limit = ?MAX_VIEW_LIMIT,
+    max_view = ?MAX_VIEW_LIMIT,
```

Comment:
Is `max_view` used just to pass around the `?MAX_VIEW_LIMIT` constant. Can just use the constant directly in that case I think?